File: rules/pascal/tests/pas.hardcoded-secret/secret.pas
program SecretTests;
begin
  password := 'supersecret123'; // crit:expect pas.hardcoded-secret
  apiKey := 'abcdef0123456789'; // crit:expect pas.hardcoded-secret
  username := 'administrator'; // crit:expect-not pas.hardcoded-secret
  secret := 'short'; // crit:expect-not pas.hardcoded-secret
end.

File: rules/csharp/tests/cs.hardcoded-secret/Secret.cs
class SecretTests {
  string password = "supersecret123";  // crit:expect cs.hardcoded-secret
  string apiKey = "abcdef0123456789";  // crit:expect cs.hardcoded-secret
  string username = "administrator";   // crit:expect-not cs.hardcoded-secret
  string secret = "short";             // crit:expect-not cs.hardcoded-secret
}
